tests/stream/stream_api.cpp

```diff
@@ -146,29 +143,59 @@ class TEST_NAME : public util::test_base {
       check_enum_class_value(sycl::stream_manipulator::flush);
 
       /** Check stream interface
-      */
+       */
       {
         auto testQueue = util::get_cts_object::queue();
         testQueue.submit([&](sycl::handler &cgh) {
-
           sycl::stream os(2048, 80, cgh);
 
           /** check get_size()
-          */
+           */
+#ifdef SYCL_CTS_ENABLE_DEPRECATED_FEATURES_TESTS
           {
             auto size = os.get_size();
-            check_return_type<size_t>(log, size,
-                                      "sycl::context::get_size()");
+            check_return_type<size_t>(log, size, "sycl::stream::get_size()");
           }
+#endif  // SYCL_CTS_ENABLE_DEPRECATED_FEATURES_TESTS
 
-          /** check get_max_statement_size()
-          */
+          /** check size()
+           */
+#ifndef SYCL_CTS_COMPILING_WITH_DPCPP
+          {
+            auto size = os.size();
+            check_return_type<size_t>(log, size, "sycl::stream::size()");
+            CHECK(noexcept(os.size()));
+          }
+#else
+          WARN(
```

Comment:
NOTE: We are about to add a new macro - SKIP. If https://github.com/KhronosGroup/SYCL-CTS/pull/469 goes first, these WARN should be replaced with SKIP.

Reply:
Acknowledged, that's a nice addition.